radix-engine/src/blueprints/resource/fungible/fungible_bucket.rs

```diff
@@ -22,7 +22,11 @@ impl FungibleBucketBlueprint {
             FungibleResourceManagerField::Divisibility.into(),
             LockFlags::read_only(),
         )?;
-        let divisibility: u8 = api.field_read_typed(divisibility_handle)?;
+        let divisibility: u8 = api
+            .field_read_typed::<FungibleResourceManagerDivisibilityFieldPayload>(
+                divisibility_handle,
```

Comment:
Am I the only one who's confused with `VersionedXyz` and `XyzFieldPayload`? Are they type aliases?

It seems that we have three types, including `Xyz` for all fields, collection keys and values (if versioned)?

Reply:
Was confusing to me initially too but `XyzFieldPayload` is just a wrapper sbor transparent struct around the `VersionedXyz`.